### PR TITLE
koord-runtime-proxy: add installation manual

### PR DESCRIPTION
**Pull request by honpey**

### CODE REVIEW

File: docs/design-archive/koord-runtime-proxy-design-doc.md [hunk 1]
```diff
@@ -117,3 +117,35 @@ plugin provider can set any hook combinations to "runtime-hooks".
...
+### Installing from packages
+TODO
+
+### Setup Kubelet
+To make koord-runtime-proxy a proxy between kubelet and containerd(dockerd), kubelet parameters should be altered as shown
+below:
+```
+kubelet <other options> --container-runtime=remote --container-runtime=/var/run/koord-runtimeproxy/runtimeproxy.sock
+```
+
```
Comment (ZYecho) [inline docs/design-archive/koord-runtime-proxy-design-doc.md:133]: for docker,kubelet need to config --docker-endpoint

File: docs/design-archive/koord-runtime-proxy-design-doc.md [hunk 2]
```diff
@@ -117,3 +117,35 @@ plugin provider can set any hook combinations to "runtime-hooks".
 
 ## Installation
 
+### Installing from sources
+get sources: git clone https://github.com/koordinator-sh/koordinator.git
```
Comment (jasonliu747) [inline docs/design-archive/koord-runtime-proxy-design-doc.md:121]: include these commands in code block

**Pull request merged**